cypress/integration/basic-functionalities/local-tests/compose/module/2_Edit_a_module.spec.js

```diff
@@ -0,0 +1,32 @@
+/// <reference types="cypress" />
+const baseURL = Cypress.env('baseURL')
+const email = Cypress.env('user').email
+const password = Cypress.env('user').password
+
+// Before running this test first run the Server test 2 in order to create a user so you can log in.
+// and after that you need to run the namespace test 1_Create_a_namespace.spec.js in order to create a namespace
+// and after that run the module test 1_Create_a_module.spec.js so you can create a module so you can use it
+describe('Test for editing a module', () => {
+  before(() => {
+    cy.login({ email, password })
+  })
+  context('Test for editing the module info and a record field', () => {
+    it('should be able to edit the module', () => {
+      cy.get('[data-test-id="button-visit-namespace"]:last').click()
+      cy.get('[data-test-id="button-admin"]').click()
+      cy.get('tr.pointer>:first-child()').click()
```

Comment:
Please, reference how we do this in other tests. And google search pointer class.